src/SystemEvent.h

```diff
@@ -0,0 +1,9 @@
+#ifndef SRC_SYSTEMEVENT_H
+#define SRC_SYSTEMEVENT_H
+
+enum SystemEvent
```

Comment:
Could you move this single value enum in SystemEventHandler.h, having a .h only for 1 enum is not so nice

Reply:
I have another branch which adds three values to this enum: `LOGGING_OFF`, `GOING_TO_SLEEP`, and `SHUTTING_DOWN`.